integ-test/src/test/java/org/opensearch/sql/sql/DateTimeFunctionIT.java

```diff
@@ -108,34 +108,34 @@ public void testAddDateWithInterval() throws IOException {
     JSONObject result =
         executeQuery("select adddate(timestamp('2020-09-16 17:30:00'), interval 1 day)");
     verifySchema(result,
-        schema("adddate(timestamp('2020-09-16 17:30:00'), interval 1 day)", null, "datetime"));
+        schema("adddate(timestamp('2020-09-16 17:30:00'), interval 1 day)", null, "timestamp"));
     verifyDataRows(result, rows("2020-09-17 17:30:00"));
 
-    result = executeQuery("select adddate(DATETIME('2020-09-16 17:30:00'), interval 1 day)");
+    result = executeQuery("select adddate(TIMESTAMP('2020-09-16 17:30:00'), interval 1 day)");
     verifySchema(result,
-        schema("adddate(DATETIME('2020-09-16 17:30:00'), interval 1 day)", null, "datetime"));
+        schema("adddate(TIMESTAMP('2020-09-16 17:30:00'), interval 1 day)", null, "timestamp"));
```

Comment:
Duplicate of above

Reply:
removed in https://github.com/Bit-Quill/opensearch-project-sql/pull/336/commits/1aec4f9ee0ca2e447f3a4b9cdff2169a98c76fcb